src/lib/checkbox/checkbox.test.ts

```diff
@@ -0,0 +1,451 @@
+/// <reference types="mocha" />
+
+import { expect } from '@esm-bundle/chai';
+import { fixture, html } from '@open-wc/testing';
+import { getShadowElement } from '@tylertech/forge-core';
+import { sendKeys, sendMouse } from '@web/test-runner-commands';
+import { spy } from 'sinon';
+import { TestHarness } from '../../test/utils/test-harness';
+import { CHECKBOX_CONSTANTS, ICheckboxComponent } from '../checkbox';
+import { IFocusIndicatorComponent } from '../focus-indicator';
+import { IStateLayerComponent } from '../state-layer';
+import exp from 'constants';
```

Comment:
Looks like we can remove this import since it was likely added by mistake (I do this all the time)